src/API/PlayerController.cpp

```diff
@@ -44,7 +44,7 @@ string PlayerController::RefreshOnline() {
 
 void PlayerController::RefreshPlatform() {
     IPlatformUserModel* userModel = NULL;
-    ::ArrayW<PlatformLeaderboardsModel *> pmarray = Resources::FindObjectsOfTypeAll<PlatformLeaderboardsModel*>();
+    /*::ArrayW<PlatformLeaderboardsModel *> pmarray = Resources::FindObjectsOfTypeAll<PlatformLeaderboardsModel*>();
```

Comment:
I thought you fixed this

Reply:
I fixed it indirectly, I put checks where this would cause crashes

Reply:
Then this should be uncommented?

Reply:
done